From e73dc9f146c7f29e7b08ddcbae3b89c6b573760f Mon Sep 17 00:00:00 2001
From: Kam Nasim <kam.nasim@windriver.com>
Date: Mon, 25 Sep 2017 16:26:54 -0400
Subject: [PATCH] US103091: IMA System Configuration

Since /tmp is now mounted on tmpfs, we will make /var/tmp as a simlink
on /tmp. Ensure that the var.tmp subdir (within /tmp), to which /var/tmp
is similinked, does not get clobbered during cleanup
---
 tmpfiles.d/tmp.conf | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/tmpfiles.d/tmp.conf b/tmpfiles.d/tmp.conf
index ffdd82f..530866b 100644
--- a/tmpfiles.d/tmp.conf
+++ b/tmpfiles.d/tmp.conf
@@ -9,10 +9,12 @@
 
 # Clear tmp directories separately, to make them easier to override
 v /tmp 1777 root root 10d
-v /var/tmp 1777 root root 30d
+v /tmp/var.tmp 1777 root root 30d
+L+ /var/tmp - - - - /tmp/var.tmp
 
 # Exclude namespace mountpoints created with PrivateTmp=yes
 x /tmp/systemd-private-%b-*
 X /tmp/systemd-private-%b-*/tmp
 x /var/tmp/systemd-private-%b-*
 X /var/tmp/systemd-private-%b-*/tmp
+X /tmp/var.tmp
-- 
1.8.3.1

